Source/Core/EllipseGeometry.js

```diff
@@ -819,7 +819,7 @@ define([
             return;
         }
 
-        ellipseGeometry._center = ellipseGeometry._ellipsoid.scaleToGeodeticSurface(ellipseGeometry._center, ellipseGeometry._center);
+        ellipseGeometry._center = ellipseGeometry._ellipsoid.scaleToGeocentricSurface(ellipseGeometry._center, ellipseGeometry._center);
```

Comment:
I don't think we want to make this change.  I'm pretty sure this will produce a different position than `scaleToGeodeticSurface`.  The problem needs to be fixed at the Entity level.